packages/block-library/src/cover/edit.js

```diff
@@ -174,6 +174,12 @@ function ResizableCover( {
 				setIsResizing( false );
 			} }
 			minHeight={ COVER_MIN_HEIGHT }
+			__experimentalShowTooltip={ true }
```

Comment:
Could we move this prop inside `__experimentalTooltipProps` object? Do we need to have a separate variable?